Include/Python.h

```diff
@@ -130,6 +130,5 @@
 #include "fileutils.h"
 #include "cpython/pyfpe.h"
 #include "cpython/tracemalloc.h"
-#include "cpython/optimizer.h"
```

Comment:
Hm. If some APIs have been exported, IMHO, adding news will be better~.

Reply:
You're right: I added a NEWS entry.